tdvt/tdvt/tabquery.py

```diff
@@ -8,18 +8,28 @@ def configure_tabquery_path():
     """Setup the tabquery path from ini settings."""
     global tab_cli_exe
     config = configparser.ConfigParser()
-    
+
     tdvt_cfg = get_ini_path_local_first('config/tdvt', 'tdvt')
     logging.debug("Reading tdvt ini file [{}]".format(tdvt_cfg))
     config.read(tdvt_cfg)
-
-    if sys.platform.startswith("darwin"):
+    tabquery_envvar = os.environ.get('TABQUERY_CLI_PATH')
+    logging.debug("TABQUERY_CLI_PATH environment variable is {}".format(tabquery_envvar))
+
+    if tabquery_envvar and not os.path.isfile(tabquery_envvar):
```

Comment:
My thought for adding this check here is that the envvar is what we look for first, but if it's wrong, we then go on to try to look for a value from a config file. Seems like it might be more forgiving for the user this way.